src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,21 @@
+import { NavLink } from "react-router-dom"
+
+function Navbar() {
+  return (
+    <header>
+        <nav>
+            <img src="" alt="" />
+        </nav>
+        <nav>
+            <ul>
+                <li><NavLink to={'/'}>Home</NavLink></li>
+                <li><NavLink to={'/films'}>Films</NavLink></li>
+                <li><NavLink to={'/series'}>Series</NavLink></li>
+                <li><NavLink to={'/books'}>Books</NavLink></li>
```

Comment:
I know this is a draft but it could be misleading having books and series too. I say let's start just with movies to simplify and build on top of it. With a couple of nav bar items is enough :)